### DISCUSSION TITLE
Use ESP32 to support Smarthome with Matter

### DISCUSSION THREAD

**Discussion by tiloio** [gcormier/megadesk#126] [General]
I would like to control my smart desk via my smart home (HomeKit, Google Home or Alexa). To do this we could use Matter (https://github.com/project-chip/connectedhomeip, https://csa-iot.org/all-solutions/matter/).

Espressif supports Matter with their ESP32 Microcontroller https://github.com/espressif/esp-matter.
The controller is wide spread and easy to program.

**Comment by gcormier** [comment]
For now the best solution would be home assistant integration using serial control.

The challenge is the size of the PCB to have enough space to play with :)
